mods/d2k/rules/world.yaml

```diff
@@ -35,6 +35,10 @@
 		Name: Corrino
 		InternalName: corrino
 		Selectable: false
+	Faction@Smuggler:
```

Comment:
Are smugglers the same as mercenaries? IIRC the AI-controlled faction allied to Ordos in the campaign had a different artwork for their heavy factory, too.